code/modules/clothing/suits/armor.dm

```diff
@@ -38,13 +38,40 @@
 			B.attackby(I,user)
 	else
 		..()
+		
+#define CLOWNABLE 0
+#define CLOWNED 1
```

Comment:
I don't see why these defines are better than a simple TRUE/FALSE on the var is_clowned. But if you really want to keep them they need to be undefined at the end of the file.

Reply:
i tried to undefine them and couldn't get it to work
literally just copied kurfursten's implementation on the guns
will just replace with true/false var